src/Sylius/Component/Review/Factory/ReviewFactoryInterface.php

```diff
@@ -21,17 +22,17 @@
 interface ReviewFactoryInterface extends FactoryInterface
 {
     /**
-     * @param mixed $subjectId
+     * @param ReviewableInterface $subject
      *
      * @return ReviewInterface
      */
-    public function createForSubject($subjectId);
+    public function createForSubject(ReviewableInterface $subject);
```

Comment:
I would do it with pleasure, but right now we use this method in our "magical" yml and when we are creating review as guest we will pass `null` instead of `Customer`, so it must stay like this until we figure out something better (not a concern of this PR).